src/util/compatibility.h

```diff
@@ -100,6 +100,29 @@ inline QScreen* getPrimaryScreen() {
     return nullptr;
 }
 
+inline
+QString uuidToStringWithoutBraces(const QUuid& uuid) {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
```

Comment:
Whats happen if uuid is null here? 
What is the difference between uuidToStringWithoutBraces and uuidToNullableStringWithoutBraces in case uuid is null?
Would be possible to join the functions?

Reply:
The tests show the expected and actual results. QUuid has no special null value, although the naming suggests this. Null is just a UUID with only zeros "000000000000-0000-0000-000000000". Unfortunately the toString() method returns them verbatim. This is usually not what you expect. That's why we neetd uuidToNullableStringWithoutBraces() in additional to the compatibility function.